client/src/components/Register/MethodTile.scss

```diff
@@ -0,0 +1,64 @@
+.mfa-method-tile {
+  flex: 25%;
```

Comment:
Why isn't this 33%

Reply:
Because 33% including margins breaks early. I'll update to `calc(33% - 1rem);`

Reply:
Try `calc(33% - (#{$tile-spacing} / 2))`